# DYSLER
## Artificial intelligence-enabled characterization of schizophrenia using risk essential biomarkers derived from transcriptomic analyses
## abstract
The reliable machine learning models based on transcriptomic-analysis to characterize schizophrenia risk is an ongoing thirst. Here, we developed a novel artificial intelligence pipeline to characterize schizophrenia from risk essential biomarkers (REBs). The pipeline was evaluated using brain RNA-sequencing data from the prefrontal cortex of 1,722 individuals. The final model characterizes schizophrenia using a 6-REB set with an area-under-the-curve of 0.77 in the validation set and 0.74 in the external test set. Annotations from large-scale genetic data and prior knowledge indicate that these REBs affect schizophrenia pathogenesis by neuro-inflammation and -development, and that schizophrenia polygenicity risk at the 6-REB set is efficient and higher than randomly selected gene sets. Moreover, animal experiments also confirmed the reliability of these REBs. Our study demonstrates the feasibility of transcriptomic-analysis-based objective characterization of schizophrenia risk, and presents preliminary evidence that our pipeline may be valuable for further understanding of schizophrenia pathogenesis and therapeutic targets.
****
![image](https://github.com/shenxiaochenn/DYSLER/blob/master/fig1.png)
****
### The overall pipeline
The pipeline consists of three components, namely the identification of significant biomarkers in SCZ-associated pathways, the capture and expansion of risk essential biomarkers (REBs), and the construction of the novel dynamic second-level learning risk (DYSLER) model. In the first step, the significant biomarkers in potential SCZ-associated pathways are obtained by traditional bioinformatics analysis. In the second step, the REBs are identified by recursive feature elimination combined with multiple machine learning models and feature engineering techniques, and then more biomarkers correlated with SCZ were further obtained based on these REBs by employing the polynomial feature derivation and the mutual information (MI) filter. In the third step, we constructed a novel DYSLER model consisting of basic learners and meta-learner based on Bayesian optimization to enhance the reliability and stability of characterizing disease risk. Briefly, we prepared a total of 8 candidate basic learners, including 2 eXtreme gradient boosting (XGBoosts), 2 gradient boosting decision trees (GBDTs), 1 random forest (RF), 1 support-vector machine (SVM), 1 k-nearest neighbor (KNN), and 1 logistic regression (LR), and two simple and well-interpretable models (RF and LR) as the candidate meta-learner. The optimization is required for finalizing the type and number of basic learners and meta-learner and their corresponding hyperparameters. When modeling DYSLER, a model fusion of those selected basic learners and meta-learner is performed by a designed stochastic process, and the results of disease risk characterization are output from the meta-learner as the final results. The above process was packaged into a whole DYSLER model with Bayesian optimization based on the tree-structured parzen estimator (TPE) algorithm.
